bus_board.rb

```diff
@@ -0,0 +1,71 @@
+require 'json'
+require 'net/http'
+require 'uri'
+
+def display_bus(bus)
+  time_until_arrival = bus["timeToStation"]/60 == 0 ? "due" : "#{(bus["timeToStation"]/60).to_s} minutes"
```

Comment:
Think splitting this up might be clearer?
```
time_until_arrival = (bus["timeToStation"]/60)
bus_time_arrival_string = time_until_arrival  == 0 ? "due" : "#{time_until_arrival} minutes"
```
Would also make it less error prone if you changed anything e.g. the name of `timeToStation`